.github/workflows/storybook.yml

```diff
@@ -0,0 +1,85 @@
+name: Storybook
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: storybook-${{ github.event.pull_request.number || github.ref }}-${{github.event.review.state}}
```

Comment:
I don't think the review.sate is needed here right, we'd end up deploying twice, once for PR opened, once for approved?